ci/variables/env

```diff
@@ -0,0 +1 @@
+DEFAULT_KITCHEN_TERRAFORM_TAG=2.0.1
```

Comment:
Same comment as https://github.com/dwp/terraform-aws-kong-gateway/pull/103#discussion_r1172247312

Reply:
removed the file